webapp/security/views.py

```diff
@@ -263,8 +273,107 @@ def api_create_notice():
 # CVE views
 # ===
 def cve_index():
-    return flask.render_template("security/cve/index.html")
+
+    # Temporary fix for data to be displayed
+    cves_query = db_session.query(CVE).first()
+    if not cves_query and os.environ["FLASK_DEBUG"] == "true":
```

Comment:
Don't do this in where. Just add an helper script somewhere that loads dummy data for you.

Reply:
Ok. Do you mean creating a `helper.py` in this folder and importing it?

Reply:
I mean create a `scripts/load_cves.py`, make it executable and run it `dotrun exec scripts/load_cves.py` whenever you want to load sample data.

Reply:
mm I added it here on purpose, so that people didn't have to do that... Because things will not work if there is no data at all

Reply:
so `/scripts/load_cves.py` in the root folder can't use `import webapp.security`